test/ParseableInterface/access-filter.swift

```diff
@@ -109,7 +109,7 @@ extension UFIProto {
 
 // CHECK: extension PublicStruct {{[{]$}}
 extension PublicStruct {
-  // CHECK: public private(set) static var secretlySettable: Int{{$}}
+  // CHECK: @_hasInitialValue public static var secretlySettable: Int
```

Comment:
We want to check that it ends in `{ get }` now, right?

Reply:
Yep, I'm working on this test now